src/GraphQL/Types/Scalars/ScalarGraphType.cs

```diff
@@ -99,7 +112,7 @@ public virtual bool CanParseValue(object value)
         /// Checks that the provided value is a valid default value.
         /// This method should not throw an exception.
         /// </summary>
-        /// <param name="value">The value to examine. Must not be <see langword="null"/>.</param>
+        /// <param name="value">The value to examine. Must not be <see langword="null"/>, as that indicates the lack of a default value.</param>
```

Comment:
> as that indicates the lack of a default value

But now `IsValidDefault(null)` yields `true` since by default `ToAST(null)` yields `new NullValue()`.

Add `if (value == null) return false` ?

Reply:
Sure.